experiment_pages/experiment/experiment_menu_ui.py

```diff
@@ -1,131 +1,153 @@
+from experiment_pages.experiment.experiment_invest_ui import InvestigatorsUI
+from databases.experiment_database import ExperimentDatabase
+
+class ExperimentMenuUI(MouserPage): #pylint: disable= undefined-variable
+    '''Experiment Menu Page Frame'''
+    def __init__(self, parent: CTk, name: str, prev_page: ChangeableFrame = None, controller: SerialPortController = None): #pylint: disable= undefined-variable
+
+        #Get name of file from file path
+        experiment_name = os.path.basename(name)
+        experiment_name = os.path.splitext(experiment_name)[0]
+        self.experiment = ExperimentDatabase(name)
+
+        super().__init__(parent, experiment_name, prev_page)
+
+        main_frame = CTkFrame(self) # pylint: disable=redefined-outer-name
+
+        main_frame.grid(row=6, column=1, sticky='NESW')
+        main_frame.place(relx=0.3, rely=0.20, relwidth= 0.40, relheight = 0.75)
+
+        main_frame.grid_columnconfigure(0, weight = 1)
+
+        self.data_page = DataCollectionUI(parent, self, name)
+        self.analysis_page = DataAnalysisUI(parent, self)
+        self.cage_page = CageConfigurationUI(name, parent, self)
+        if controller is None:
+            self.rfid_page = MapRFIDPage(name, parent, self)
+        else:
+            self.rfid_page = MapRFIDPage(name, parent, self, controller)
+        self.invest_page = InvestigatorsUI(parent, self)
+
+        button_size = 30
+
+
+        self.collection_button = CTkButton(main_frame, text='Data Collection', width=button_size,
+                                command= self.data_page.raise_frame)
+        self.analysis_button = CTkButton(main_frame, text='Data Analysis', width=button_size,
+                                command= self.analysis_page.raise_frame)
+        self.group_button = CTkButton(main_frame, text='Group Configuration', width=button_size,
+                                command= lambda: [self.cage_page.raise_frame(),
+                                                  self.cage_page.update_controller_attributes(),
+                                                  self.cage_page.update_config_frame()])
+        self.rfid_button = CTkButton(main_frame, text='Map RFID', width=button_size,
+                                command=  self.rfid_page.raise_frame)
+
+        self.collection_button.grid(row=0, column=0, ipady=10, ipadx=10, pady=10, padx=10)
+        self.analysis_button.grid(row=1, column=0, ipady=10, ipadx=10, pady=10, padx=10)
+        self.group_button.grid(row=2, column=0, ipady=10, ipadx=10, pady=10, padx=10)
+        self.rfid_button.grid(row=3, column=0, ipady=10, ipadx=10, pady=10, padx=10)
+
+
+        if self.menu_button:
+            self.menu_button.destroy()
+
+        self.bind("<<FrameRaised>>", self.on_show_frame)
+
+         # Assume buttons are enabled initially- This allows for them to be disabled before RFID Mapping is done
+        self.disable_buttons_if_needed()

```

Comment:
Good comment makes it easier to understand a change that I otherwise would've overlooked